content/2021/January/README.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "January PythonPune meetup 2021"
+date: 2021-01-31T23:16:07+05:30
+event_date: 2021-01-23T11:00:00+05:30
+url: event/january-pythonpune-meetup-2021
+author: Bhavin Gandhi
+featured_image:
+categories:
+- Event
+tags:
+- Python Internals
+---
+
+  * [Event page on meetup.com](https://www.meetup.com/PythonPune/events/275599702/)
+  * [Community Newsletter](./community_news.md)
+  * [Recording](https://youtu.be/cEYwfeAXGwQ)
+
+## Context Managers: Manage your resources smartly
+  * **Speaker**: Pradhvan Bisht ([blog](https://pradhvanbisht.in),
+    [Twitter](https://twitter.com/BishtPradhvan))
+  * **Description**: In this talk Pradhvan sheds light on the context
+    managers in Python. He covers things like the `with` statement,
+    creating your own context manager and the ` @contextmanager`
+    decorator. He also shows some real-life examples of context
+    managers.
+  * **References**:
+    * [Slides](http://bit.ly/ContextManagersSlides)
+    * [Blog post](https://pradhvan.scrollstack.com/post/2771/Context-Managers-in-Python)
+    * [Video recording](https://youtu.be/6x1AoF8yq8o)
```

Comment:
This video is private. If you are still working on it and plan to make it public then it's OK to me.

Reply:
It should be available now.